source/2015-06-12-what-happens-in-kalamata.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+title: What happens in Kalamata
+date: 2015-06-12 12:47 UTC
+tags:
+---
+Panter im Klassenalger 2015
+
+Jedes Jahr gibt es für die panter Mitarbeiter die Möglichkeit nach Griechenland
+ins Klassenlager zu fahren.
+
+Wieso man als Mitarbeiter eine Woche Ferien opfern sollte um mit den anderen
+Mitarbeitern (aktuell leider noch ohne Mitarbeiterinnen) nach Griechenland zu
+fahren ist vielleicht am Anfang noch ein Rätsel. Die Kommentare von anderen
+Mitarbeitern wie „cool“ das sei, lässt allerdings nicht viel Zweifel aufkommen,
+dass man die Gelegenheit nicht missen sollte.
+
+Ende Mai, war’s dann endlich soweit, mein erstes Mal – Klassenlager
+natürlich. Schon der Check-in am Flughafen begann voller Vorfreude. Die
+Security Control hatte uns oder wohl eher unsere ganze mitgeschleppte
+Technologie ebenfalls besonders gern, sodass gewisse Taschen gleich viermal
+durch den Scanner durften. Dass verhiess ja schon mal viel Gutes. Das erste
+Gebot stand bereits in der Informationsmail – es durfte an keinen alltäglichen
+Arbeitsprojekten gearbeitet werden.
+
+Angekommen in Athen ging’s per Auto nach einem Zwischenstopp in Korinth nach
```

Comment:
both is correct actually according to http://www.reden-und-praesentieren.de/beitraege.php?beitrag_id=1780
war's is the preferred option though.